IdentityCore/src/cache/key/MSIDDefaultCredentialCacheQuery.m

```diff
@@ -183,4 +183,9 @@ - (NSString *)queryClientId
     return nil;
 }
 
+- (BOOL)isShared
+{
+    return (self.credentialType == MSIDRefreshTokenType);
```

Comment:
nit: query inherits from key, no need to override this in both of them

Reply:
Done.